pom.xml

```diff
@@ -863,5 +863,16 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <!-- maven-compiler-plugin -->
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
I don't think we are proposing to ship jdk17 versions of Narayana, so I am not clear what setting the maven.compiler.target to 17 achieves for the narayana artifacts that would be accessible on maven central?